rotkehlchen/db/schema.py

```diff
@@ -733,6 +733,9 @@
 );
 """
 
+DB_CREATE_INDEXES = """
+CREATE INDEX IF NOT EXISTS idx_receipt_log_topics_topic_id ON evmtx_receipt_log_topics(topic_id);
```

Comment:
the question with this index then becomes how big is it? And does it then make sense to have the initial change? As the initial change was just to reduce DB size.